### PR TITLE
Add workload management annotation

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

**Review by vitus133 — APPROVED** [review]

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Comment by kquinn1204** [comment]
@vitus133 can you review these annotation changes or get someone to review as we in docs need that LGTM before we can merge. It is part of our defined process for changes like this.

**Pull request merged**
